docker-build.sh

```diff
@@ -64,13 +68,21 @@ buildDockerContainer()
 
   writeConfigToFile
 
-  ${BUILD_CONFIG[DOCKER]} build -t "${BUILD_CONFIG[CONTAINER_NAME]}" -f "${dockerFile}" . --build-arg "OPENJDK_CORE_VERSION=${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" --build-arg "HostUID=${UID}"
+  ${BUILD_CONFIG[DOCKER]} ${BUILD_CONFIG[USE_DOCKER]} build -t "${BUILD_CONFIG[CONTAINER_NAME]}" -f "${dockerFile}" . --build-arg "OPENJDK_CORE_VERSION=${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" --build-arg "HostUID=${UID}"
 }
 
 # Execute the (Adoptium) OpenJDK build inside the Docker Container
 buildOpenJDKViaDocker()
 {
 
+  DOCKER_PPODMAN="${1}"
```

Comment:
Is this definition used anywhere? And if so is it supposed to have `PP`?

Reply:
yah, rmeoved...